.github/actions/external_runner/action.yml

```diff
@@ -42,7 +42,7 @@ runs:
 
     - uses: actions/setup-go@v3
       with:
-        go-version: '^1.18.8'
+        go-version: '^1.19.3'
```

Comment:
Go 1.19.5 includes vuln fixes:

go1.19.5 (released 2023-01-10) includes fixes to the compiler, the linker, and the crypto/x509, net/http, sync/atomic, and syscall packages. See the [Go 1.19.5 milestone](https://github.com/golang/go/issues?q=milestone%3AGo1.19.5+label%3ACherryPickApproved) on our issue tracker for details.

I think we could directly bump to it. wdyt?